src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -298,6 +290,22 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
     display: none;
     visibility: hidden;
   }
+
+  .pf-m-chip-container {
```

Comment:
I should have caught this before, but I think here and in a few other places in this file, we should make sure the modifier selector applies to the element it modifies, since we reuse modifier class names. `.pf-m-chip-container` could be a modifier from another component that would inherit styles we don't want here. I think we updated `.pf-m-hidden` in this way here https://github.com/patternfly/patternfly-next/blob/95ed2defabe187ccb6898cdecbb19a0cc07230ff/src/patternfly/components/DataToolbar/data-toolbar.scss#L241-L251

Can we update `.pf-m-modifier`, `.pf-m-label` ([link](https://github.com/patternfly/patternfly-next/blob/95ed2defabe187ccb6898cdecbb19a0cc07230ff/src/patternfly/components/DataToolbar/data-toolbar.scss#L289)), and the modifiers in the SCSS loops?